modules/ocf_mirrors/manifests/init.pp

```diff
@@ -45,6 +47,10 @@
       source => 'puppet:///modules/ocf_mirrors/README.html',
       owner  => mirrors,
       group  => mirrors;
+
+    '/usr/local/sbin/record-mirrors-stats':
+      content => template('ocf_mirrors/record-mirrors-stats.py.erb'),
+      mode    => '0644';
```

Comment:
it now uses environment variables, seeded by cron. The password is stored in the root crontab so it should be secure enough + no need to add another config file. Does that seem reasonable?